app/controllers/avo/associations_controller.rb

```diff
@@ -109,7 +109,11 @@ def set_attachment_class
     end
 
     def set_attachment_resource
-      @attachment_resource = @field.use_resource || (App.get_resource_by_model_name @attachment_class)
+      @attachment_resource = if @field.present? && @field.respond_to?(:use_resource) && @field.use_resource.present?
+        @field.use_resource
+      else
+        App.get_resource_by_model_name @attachment_class
+      end
```

Comment:
I think all associations fields respond to `use_resource` but it can return `nil`

This validations of `@field.present? && @field.respond_to?(:use_resource) && @field.use_resource.present?` are silencing some errors that maybe we don't want to silence. 

Is there any moment that we want `associations_controller` called without a `@field`?!

Reply:
> Is there any moment that we want associations_controller called without a @field?!

Good question! I guess not.

This is a good place to raise an error with a proper message. "We couldn't find the `X` association." or "We couldn't find a resource for `X` association. Are you sure you created one?"

So the users understand what happened.